rbd.c

```diff
@@ -238,9 +251,10 @@ static int tcmu_rbd_service_register(struct tcmu_device *dev)
 }
 
 #ifdef RBD_LOCK_ACQUIRE_SUPPORT
-static void tcmu_rbd_service_status_update(struct tcmu_device *dev,
+static int tcmu_rbd_service_status_update(struct tcmu_device *dev,
 					   bool has_lock)
 {
+	return 0;
```

Comment:
Squash "rbd: remove possible warning" commit into an earlier commit that made the void -> int change.

Reply:
I have split this into a new PR in https://github.com/open-iscsi/tcmu-runner/pull/673.

And also have split some other commits into separate PRs to be more easy to review: https://github.com/open-iscsi/tcmu-runner/pull/672, https://github.com/open-iscsi/tcmu-runner/pull/671 and https://github.com/open-iscsi/tcmu-runner/pull/670.

Please take a look, thanks.